src/test/components/__snapshots__/FlameGraph.test.js.snap

```diff
@@ -253,12 +253,11 @@ exports[`FlameGraph matches the snapshot 1`] = `
     <li
       class="filterNavigatorBarItem filterNavigatorBarRootItem filterNavigatorBarSelectedItem filterNavigatorBarLeafItem"
       data-index="0"
-      title="Complete \\"Empty\\""
```

Comment:
Most of the titles are removed now, but they were duplicate and redundant anyway.